Description: disable digest 0.9 support
 digest 0.9 has been dropped from Debian, so we no longer want to build support
 for (and introduce dependencies on) it
Author: Noah Meyerhans <noahm@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: hmac-sha256/Cargo.toml
===================================================================
--- hmac-sha256.orig/Cargo.toml
+++ hmac-sha256/Cargo.toml
@@ -43,17 +43,10 @@ features = ["oid"]
 optional = true
 package = "digest"
 
-[dependencies.digest09]
-version = "0.9.0"
-optional = true
-package = "digest"
-
 [features]
 default = []
 opt_size = []
 traits = [
-    "traits09",
     "traits010",
 ]
 traits010 = ["digest010"]
-traits09 = ["digest09"]
